gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -678,6 +797,18 @@ def _prepare_device_data(self, device_info):
         device['advance_sharing_interfaces'] = (
                     self._get_advance_sharing_interfaces(device['id']))
         return device
+    
+    def device_healthy(self, event):
+        nfp_context = event.data['nfp_context']
+
+        device = nfp_context['network_function_device']
+
+        self._update_network_function_device_db(device, 'HEALTH_CHECK_COMPLETED')
+
+        self._create_event(event_id='DEVICE_ACTIVE', event_data=nfp_context)
+
+        self._create_event(event_id='CREATE_DEVICE_CONFIGURATION',
+                               event_data=nfp_context, is_internal_event=True)
```

Comment:
Yes, i think in orchestrator needs a different sort of handling now. Not everything needs to be serialized because it makes error conditions simple, we might have to think and design that when paralell events happen and if one of them fail how do we handle. There must be some generic design to handle such situations not making things sequenced. We need to think through, like - there are two events which are related and executes paralelly and can finish separately, and if one of them fails, the overall result should be fail.
From monday, plan is to run over all error conditions and do the required design and coding.